scanapp/tasks.py

```diff
@@ -24,9 +24,11 @@
 from __future__ import absolute_import, unicode_literals
 
 import json
+import logging
 import requests
 import subprocess
 
+from giturl import *
```

Comment:
It was given in the documentation of giturl. 
https://github.com/willyg302/giturl#usage